<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UILinuxJoystickControls</class>
 <widget class="QWidget" name="UILinuxJoystickControls">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>498</width>
    <height>334</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Tracker settings</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../gui/images/opentrack.png</normaloff>../gui/images/opentrack.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>12</number>
   </property>
   <property name="topMargin">
    <number>6</number>
   </property>
   <property name="rightMargin">
    <number>12</number>
   </property>
   <property name="bottomMargin">
    <number>6</number>
   </property>
   <item>
    <widget class="QFrame" name="frame">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Device</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="joylist">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Mapping</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="1">
       <widget class="QComboBox" name="joy_1">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="currentIndex">
         <number>1</number>
        </property>
        <item>
         <property name="text">
          <string>Disabled</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Joystick axis #1</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Joystick axis #2</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Joystick axis #3</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Joystick axis #4</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Joystick axis #5</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Joystick axis #6</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Joystick axis #7</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Joystick axis #8</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="joy_2">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="currentIndex">
         <number>2</number>
        </property>
        <item>
         <property name="text">
          <string>Disabled</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Joystick axis #1</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Joystick axis #2</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Joystick axis #3</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Joystick axis #4</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Joystick axis #5</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Joystick axis #6</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Joystick axis #7</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Joystick axis #8</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="joy_3">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="currentIndex">
         <number>3</number>
        </property>
        <item>
         <property name="text">
          <string>Disabled</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Joystick axis #1</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Joystick axis #2</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Joystick axis #3</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Joystick axis #4</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Joystick axis #5</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Joystick axis #6</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Joystick axis #7</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Joystick axis #8</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QComboBox" name="joy_4">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="currentIndex">
         <number>4</number>
        </property>
        <item>
         <property name="text">
          <string>Disabled</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Joystick axis #1</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Joystick axis #2</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Joystick axis #3</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Joystick axis #4</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Joystick axis #5</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Joystick axis #6</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Joystick axis #7</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Joystick axis #8</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QComboBox" name="joy_5">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="currentIndex">
         <number>5</number>
        </property>
        <item>
         <property name="text">
          <string>Disabled</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Joystick axis #1</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Joystick axis #2</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Joystick axis #3</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Joystick axis #4</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Joystick axis #5</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Joystick axis #6</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Joystick axis #7</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Joystick axis #8</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QComboBox" name="joy_6">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="currentIndex">
         <number>6</number>
        </property>
        <item>
         <property name="text">
          <string>Disabled</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Joystick axis #1</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Joystick axis #2</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Joystick axis #3</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Joystick axis #4</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Joystick axis #5</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Joystick axis #6</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Joystick axis #7</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Joystick axis #8</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>X</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Y</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Z</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Yaw</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Pitch</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Roll</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections/>
 <slots>
  <slot>startEngineClicked()</slot>
  <slot>stopEngineClicked()</slot>
  <slot>cameraSettingsClicked()</slot>
 </slots>
</ui>
